controllers/storage/minio.go

```diff
@@ -41,21 +51,30 @@ func (m *MinIOReconciler) Reconcile() (*lcm.CRStatus, error) {
 		return m.ProvisionExternalStorage()
 	}
 
-	err := m.KubeClient.Get(m.Ctx, m.getminIONamespacedName(), &minioCR)
+	err := m.KubeClient.Get(m.getMinIONamespacedName(), &minioCR)
 	if k8serror.IsNotFound(err) {
-		// TODO need test
 		return m.Provision()
 	} else if err != nil {
 		return minioNotReadyStatus(ErrorReason0, err.Error()), err
 	}
 
+	var minioStatefulSet appsv1.StatefulSet
+	err = m.KubeClient.Get(m.getMinIONamespacedName(), &minioStatefulSet)
+	if err != nil {
+		return minioNotReadyStatus(ErrorReason0, err.Error()), err
```

Comment:
What does `ErrorReason0` means? The naming is too fuzzy.

Reply:
fixed